package.json

```diff
@@ -26,10 +26,12 @@
     "babel-plugin-jsx": "1.2.0",
     "babel-preset-es2015": "6.6.0",
     "babel-preset-react": "6.5.0",
+    "bootstrap": "3.3.6",
```

Comment:
FYI we already have a link to the bootstrap CSS in `base.html` somewhere. Feel free to ignore for this PR but eventually we should switch to using whatever comes with this library